.java-version

```diff
@@ -0,0 +1 @@
+1.8
```

Comment:
Our default java environment is JDK 11. Adding this marker will automatically switch javac to 1.8 (which is required by AGP 3.6) is the developer is using `jenv`.

Reply:
Cool, we don't use jenv, but it's a simple dot-file, so sgtm